### DISCUSSION TITLE
Slack update notifications could be better

### DISCUSSION THREAD

**Discussion by kamal0808** [outline/outline#2762] [General]
The document update notifications on slack right now only show the top line in slack message preview. It could be improved to also show what has been updated by highlighting the changes in the message preview. 
It could also be achieved by updating the documents by writing new content at the top instead of bottom, but nobody follows that.

![141071DF-D4F5-47A8-855B-4ECE47012F33](https://user-images.githubusercontent.com/15616412/141470057-395bb2f2-e7d9-47bf-be30-81c2d782c254.jpeg)

**Comment by daladim** [comment]
Since 0.67, the history highlights the diffs in the page. Maybe this would make this idea easy to implement now?

**Reply by daladim** [reply]
That's probably a duplicate of my #4695 then
